requirements.txt

```diff
@@ -3,7 +3,7 @@ APIFlask==1.3.0
 apispec==6.3.0
 APScheduler==3.10.1
 attrs==22.2.0
-backports.zoneinfo==0.2.1
+backports.zoneinfo==0.2.1;python_version<"3.9"
```

Comment:
Is there any way to make pip freeze to generate it automagically?

Reply:
I don't think so, but a solution may be to have a requirements-dev.txt where each python versions chooses its required packages.